browser/brave_content_browser_client.h

```diff
@@ -61,6 +61,18 @@ class BraveContentBrowserClient : public ChromeContentBrowserClient {
       mojo::PendingRemote<network::mojom::URLLoaderFactory>* out_factory)
       override;
 
+  bool AllowWorkerFingerprinting(
+      const GURL& url,
```

Comment:
pls consider using `Origin`s, and convert to `GURL` only where necessary. Logically you're doing checks per origin and not per URL. Chromium moves to `Origin` in similar scenarios.

Reply:
This really isn't feasible until we migrate more of our own code. For example, `AllowWorkerFingerprinting` calls `WorkerGetBraveFarblingLevel` which calls `brave_shields::GetBraveShieldsEnabled` which takes a GURL.